diesel_codegen_shared/src/schema_inference/pg.rs

```diff
@@ -2,6 +2,98 @@ use diesel::*;
 use diesel::pg::PgConnection;
 use std::error::Error;
 
+use super::data_structures::*;
+
+table! {
+    pg_attribute (attrelid) {
```

Comment:
maybe add a link to <https://www.postgresql.org/docs/9.5/static/catalog-pg-attribute.html> since the field names are pretty weird (same for the next 3 comments or so)